src/columntable.jl

```diff
@@ -95,3 +95,39 @@ function withinitem(nitem, df; tag = 'I')
     value.item = repeat(PooledArray(nlevels(nitem, tag=tag)), outer=nrow)
     value
 end
+
+"""
+    power_table(sim, 0.05)
+
+Returns a `DataFrame` with two columns, `coefname` and `power`, with the proportion of 
+simulated p-values less than alpha, for `sim`, the output of `simulate_waldtests`.
+"""
+
+function power_table(sim, alpha = 0.05)
+    pvals = DataFrame(columntable(sim).p)
+    pvals = stack(pvals, 1:ncol(pvals)) 
+    pwr = by(pvals, :variable, :value => x->mean(x.<alpha) )
+    rename!(pwr, ["coefname", "power"])
+end
+
+"""
+    sim_to_df(sim)
+
+Returns a `DataFrame` with 6 columns: `iteration`, `coefname`, `beta`, `se`, `z`, `p`. 
+Rows are all the coefficients for each iteration of `sim`, the output of `simulate_waldtests`.
```

Comment:
Do we want to include `iteration` as a column? `simulate_waldtests` will always return the same values for a given RNG, but the ordering is not guaranteed when multi-threading.

Reply:
You need iteration just to group the rows so that the coefficients from each run are kept together (e.g., you might want to count what proportion of runs simulating all null fixed effects have at least 1 significant effect). The numeric id itself is arbitrary.

Reply:
Add that bit of info to the doc string? That the `iteration` is not guaranteed to be the same across runs even though the samples will be.